changelog.txt

```diff
@@ -1,3 +1,295 @@
+Change: Standardise unit conversions and allow decimal places (#10795)
+Change: Use separate names for default stations/roadstops (#10786)
+Change: [MacOS] Require at least 10.15 to run the game (#10745)
+Change: Hide all variants from UI when (display) parent is hidden (#10708)
+Change: Split Game options into General, Graphics and Sound tabs (#10674)
+Change: Extend entity override manager and station spec lists to support 16 bit IDs (#10672)
+Change: Base autosaves intervals on real time (instead of game time) (#10655)
+Change: Allow overbuilding station and waypoint tiles (#10618)
+Change: Use realtime for Linkgraph update settings (#10610)
+Change: Make tick length 27 milliseconds (#10607)
+Change: Increase max cargo age and let min cargo payment approach zero (#10596)
+Change: Show buy company dialog window even when playing in the AI company (#10459)
+Change: Use HTTPS for content-service connections (#10448)
+Change: Log AI/GS Squirrel crashes in white text for readability (#10375)
+Change: Big UFO disaster targets current location of a random train (#10290)
+Change: Remove land generator setting from World Generation GUI (#10093)
+Change: Build signals to the next junction when dragging regardless of the Ctrl state (#9637)
+Change: Allow dedicated server to use threaded saves (#10787)
+Change: [NewGRF] Increase vehicle random data from 8 to 16 bits (#10701)
+Change: [NewGRF] Read Action 3 IDs as extended-bytes for all features (#10672)
+Change: [NewGRF] Make Action 3 debug messages more consistent (#10672)
+Change: [NewGRF] Extend callback 161 (engine name) with bit 0x22 for context 'Autoreplace - Vehicles in use' (#10666)
+Change: [Script] Replace easy/medium/hard values with default value (#11959)
+Change: [Script] Limit total script ops that can be consumed by a list valuate (#11670)
+Change: [Script] Allow GS access to ScriptGroup, ScriptGameSettings.IsDisabledVehicleType, more ScriptCompany and more ScriptOrder functions (#10642)
+Change: [Script] Improve ScriptText validation error messages (#10545)
+Change: [Script] Restore support of {RAW_STRING} in ScriptText (#10492)
+Change: [Script] Validate ScriptText parameters type and amount (#10492)
+Change: [Script] Automate the ScriptObject reference counting (#10492)
+Change: [Script] Extract params info from GS strings (#10492)
+Change: [Script] A ScriptText with too many parameters is now a fatal error (#10483)
+Fix #11918: Houses should only build next to road stops, not any station type (#11919)
+Fix #11827: Make text layouter aware of ligatures (#11831)
+Fix #11752: Characters could be repeated when wrapping multi-line text (#11761)
+Fix #11748: Decreasing service interval value sufficiently would result in it wrapping around (#11749)
+Fix #11696: Graphics set parameters missing from survey data (#11764)
+Fix #11629: Crash when getting the nearest town for rotated airports (#11631)
+Fix #11618: Don't try to display unavailable zoom levels in sprite aligner (#11619)
```

Comment:
```suggestion
```
There's  feature to show zoom levels in the aligner, so this likely wasn't in 13.